osmoutils/accum/accum_helpers.go

```diff
@@ -7,20 +7,20 @@ import (
 )
 
 // Creates a new position at accumulator's current value with a specific number of shares and unclaimed rewards
-func createNewPosition(accum AccumulatorObject, addr sdk.AccAddress, numShareUnits sdk.Dec, unclaimedRewards sdk.DecCoins, options *Options) {
+func createNewPosition(name AccumulatorObject, index string, numShareUnits sdk.Dec, unclaimedRewards sdk.DecCoins, options *Options) {
```

Comment:
Trying to reconcile this renaming with the other functions and tests – here you call the `AccumulatorObject` `name`, whereas in `AddToPosition` and in tests you call the address string `name`. We should probably align these before merging

Reply:
Second this

Reply:
Erroneous change. Fixed this and other name issues found while re-reviewing